alluxio-ui/common/public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
Are the index files necessary? They look like placeholders

Reply:
They are placeholders, but also necessary since react uses them as templates. Specifically, '<div id="root"></div>' is where the react app will be bound.